.github/workflows/test.yml

```diff
@@ -134,13 +134,6 @@ jobs:
           - '16.0'
           - '14'
           - '14.0'
-          - '12'
```

Comment:
Are some of these changes because you originally used some of the commits from `dev/4.x` and then retargetted main?

Reply:
That's right. I'm going to add them again